src/solc_0.7/AssetGiveaway/ClaimERC1155.sol

```diff
@@ -0,0 +1,78 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.7.1;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "../common/Interfaces/AssetToken.sol";
```

Comment:
maybe we should use ERC1155 interface. I think openzeppelin have even one

Reply:
Great will check